runtime/src/main/java/com/datastax/oss/quarkus/runtime/internal/health/CassandraHealthCheck.java

```diff
@@ -21,14 +21,12 @@
 import java.util.HashMap;
 import java.util.Map;
 import javax.annotation.PostConstruct;
-import javax.enterprise.context.ApplicationScoped;
 import org.eclipse.microprofile.health.HealthCheck;
 import org.eclipse.microprofile.health.HealthCheckResponse;
 import org.eclipse.microprofile.health.HealthCheckResponseBuilder;
-import org.eclipse.microprofile.health.Readiness;
 
-@Readiness
-@ApplicationScoped
+// @Readiness
```

Comment:
Is this temporary? Should we remove this class?

Reply:
I think that we can remove this health-check entirely. Done in https://github.com/datastax/cassandra-quarkus/pull/127/commits/5a31d0cb19096909e9e0837493c719f3867a7bde